lib/liquider/liquid.y

```diff
@@ -73,7 +73,7 @@ rule
   AdditiveExpression
   : MultiplicativeExpression
   | AdditiveExpression PLUS MultiplicativeExpression { result = Ast::BinOpNode.new(:+, val[0], val[2]) }
-  | AdditiveExpression MINUS MultiplicativeExpression { result = Ast::BinOpNode.new(:-, val[0], val[2], :-) }
```

Comment:
The tests only test the values, so if `left` is `:+` on both sides, they're equal